autopush/router/simple.py

```diff
@@ -159,9 +158,20 @@ def route_notification(self, notification, uaid_data):
             returnValue(self.delivered_response(notification))
         else:
             self.metrics.increment("router.broadcast.miss")
-            returnValue(self.stored_response(notification))
-
-    #############################################################
+            retVal = self.stored_response(notification)
+            if self.udp is not None and "server" in self.conf:
+                # Attempt to send off the UDP wake request.
+                try:
+                    yield requests.post(
```

Comment:
Er, I dunno why this works in the tests. yield can only be done on results that are deferred, requests.post is a blocking call that doesn't return a yield. It needs a deferToThread in front.